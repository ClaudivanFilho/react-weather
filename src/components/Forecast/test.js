import React from 'react';
import {shallow} from 'enzyme';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-15';
import renderer from 'react-test-renderer';

import { Forecast } from './index';

configure({ adapter: new Adapter() });

const forecastMock = {
  forecast: {
    list: []
  }
}

test('Test Forecast Component', () => {
  const component = shallow(<Forecast forecast={forecastMock}/>);
  expect(component.text()).toEqual('<CSSTransitionGroup />');
});

test('Test Forecast Snapshot', () => {
  const component = renderer.create(
    <Forecast forecast={forecastMock}/>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})