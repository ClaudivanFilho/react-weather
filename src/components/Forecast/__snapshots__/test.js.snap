// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Forecast Snapshot 1`] = `
<span>
  <div
    className="col-sm-12 float-left"
  >
    <div
      className="col-sm-12 ui horizontal divider"
    >
      Forecast
    </div>
    0
  </div>
</span>
`;
